fmriprep/workflows/base.py

```diff
@@ -237,7 +237,7 @@ def init_single_subject_wf(subject_id):
     # Preprocessing of T1w (includes registration to MNI)
     anat_preproc_wf = init_anat_preproc_wf(
         bids_root=str(config.execution.bids_dir),
-        debug=config.execution.debug is True,
+        debug=config.execution.sloppy is True,
```

Comment:
might as well clean this up while we're touching it.
```suggestion
        debug=config.execution.sloppy,
```